cranelift/codegen/src/machinst/pcc.rs

```diff
@@ -6,8 +6,12 @@ use crate::trace;
 
 pub(crate) fn get_fact_or_default<I: VCodeInst>(vcode: &VCode<I>, reg: Reg, width: u16) -> Fact {
     trace!(
-        "get_fact_or_default: reg v{} -> {:?}",
-        reg.to_virtual_reg().unwrap().index(),
+        "get_fact_or_default: reg {} -> {:?}",
+        if reg.is_virtual() {
+            format!("v{}", reg.to_virtual_reg().unwrap().index())
+        } else {
+            format!("r{}", reg.to_real_reg().unwrap().hw_enc())
+        },
```

Comment:
I think this can be simplified to

```suggestion
        "get_fact_or_default: reg {reg:?} -> {:?}"
```